packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -821,8 +825,8 @@ class _CupertinoModalPopupRoute<T> extends PopupRoute<T> {
     assert(_animation == null);
     _animation = CurvedAnimation(
       parent: super.createAnimation(),
-      curve: Curves.ease,
-      reverseCurve: Curves.ease.flipped,
+      curve: Curves.linearToEaseOut,
```

Comment:
Describe how this was derived in a comment (just saying eyeballed is fine). This will help the next maintainer judge how hard the previous person tried to make this perfect and how much confidence they should have in trying to improve this.

Reply:
Done